templates/default/ckan_properties.ini.erb

```diff
@@ -177,8 +177,8 @@ ckan.resource_proxy.chunk_size = 8192
 # package_hide_extras = for_search_index_only
 #package_edit_return_url = http://another.frontend/dataset/<NAME>
 #package_new_return_url = http://another.frontend/dataset/<NAME>
-#ckan.recaptcha.publickey =
-#ckan.recaptcha.privatekey =
+ckan.recaptcha.publickey = ${ssm:/config/CKAN/<%= node['datashades']['version'] %>/app/<%= node['datashades']['app_id'] %>/recaptcha.publickey}
```

Comment:
Do we need to do any check for null here just in case the values are not set?

Reply:
the code is already checking for blank. I have already introduced the keys to all environments in preparations for config enablement from param store.